HACKING.md

```diff
@@ -121,10 +121,57 @@ have written. Some rules:
 Most code is shared as-is between open source and the internal Meta version of
 Buck2. However, there are some exceptions:
 
-* The open-source remote execution client is different, because our internal
-  one works with custom servers/infrastructure that is not publicly available.
-* There are places controlled with `is_open_source()` which change configuration
+- The open-source remote execution client is different, because our internal one
+  works with custom servers/infrastructure that is not publicly available.
+- There are places controlled with `is_open_source()` which change configuration
   between the internal and open source versions.
-* Some places use `@oss-enable` or `@oss-disable` to comment/uncomment lines
-  of code. The internal code is visible, but the comment markers are moved
-  during export/import of code.
+- Some places use `@oss-enable` or `@oss-disable` to comment/uncomment lines of
+  code. The internal code is visible, but the comment markers are moved during
+  export/import of code.
```

Comment:
Ugh. This was supposed to only change `*` to `-` in the bullet points but of course my editor reflowed it on save. I can back this out if desired.